pkg/rpc/provider/blockchain.go

```diff
@@ -120,7 +120,7 @@ func (p *RpcProvider) BlockResults(ctx context.Context, height *int64) (*coretyp
 // Commit implements client.CometRPC.
 func (p *RpcProvider) Commit(ctx context.Context, height *int64) (*coretypes.ResultCommit, error) {
 	heightValue := p.normalizeHeight(height)
-	header, data, err := p.adapter.Store.GetBlockData(ctx, heightValue)
+	header, data, err := p.adapter.RollkitStore.GetBlockData(ctx, heightValue)
```

Comment:
could you open an issue, this block structure is not the one we want to use. the header is not compatible with ibc here